staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/types_jsonschema.go

```diff
@@ -117,6 +117,19 @@ type JSONSchemaProps struct {
 	// Defaults to atomic for arrays.
 	// +optional
 	XListType *string `json:"x-kubernetes-list-type,omitempty" protobuf:"bytes,42,opt,name=xKubernetesListType"`
+
+	// x-kubernetes-map-type annotates a map to further describe its topology.
+	// This extension must only be used on maps and may have 2 possible values:
+	//
+	// 1) `map`:
+	//      These maps are actual maps (key-value pairs) and each fields are independent
+	//      from each other (they can each be manipulated by separate actors). This is
+	//      the default for all maps.
+	// 2) `atomic`: the list is treated as a single entity, like a scalar.
+	//      Atomic maps will be entirely replaced when updated.
+	// Defaults to map for maps.
```

Comment:
This is also false for list-type: we don't default these. We have implicit default semantics, not default values. Would drop this line.